tests/unit_tests/pyanaconda_tests/modules/localization/test_module_localization.py

```diff
@@ -232,6 +233,9 @@ def test_locale_data(self):
         }
         assert data == english_us
 
+    def test_common_locales(self):
+        assert self.localization_interface.GetCommonLocales() == langtable.list_common_locales()
```

Comment:
This test should confirm that we are not trying to send data that are not supported by DBus. Please, make sure that the return value is a list and check if, for example, `en_US.UTF-8` and `ja_JP.UTF-8` is in this list. I would be fine with checking all values. If langtable starts to return different values, it will probably break the Web UI pixel tests anyway.